.github/workflows/tests.yml

```
@@ -59,6 +59,8 @@ jobs:
       - name: Run tests with coverage
         run: make coverage
 
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)